docs/one-click-integrations/github-integration.md

```diff
@@ -120,6 +120,13 @@ The table below lists the data that the GitHub integration collects from your Gi
     </td>
     <td><a href="../../metrics/lead-time-reviews/">Lead time and reviews dashboard</a>,<br/><a href="../../metrics/work-in-progress/">Work in progress dashboard</a></td>
 </tr>
+<tr>
+    <td>Teams</td>
+    <td>
+        -
```

Comment:
Will the CLI and Ingestion API also allow specifying a team? :thinking:

Reply:
This pull request answers the question:

https://github.com/codacy/pulse/pull/669